### PR TITLE
InvestigationSummaryParse

### PR DESCRIPTION
**Pull request by dorschw**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
closes: https://github.com/demisto/etc/issues/46492
closes: https://github.com/demisto/etc/issues/46368

## Description
Parsing EDR & other results into context automatically

## Minimum version of Cortex XSOAR
- [ ] 6.1.0

## Does it break backward compatibility?
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MalwareParser/Scripts/Parse/MalwareFindings.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        if signer := demisto.get(self.context, 'MicrosoftATP.File.Signer'):
+            self.additional_attributes['Signer'] = signer
+            self.sources.add(Source.TODO)  # todo
+
+        # WildFire
+        if publisher := demisto.get(self.context, 'File.DigitalSignature.Publisher'):
+            self.status = Status.BENIGN
+            self.sources.add(Source.TODO)  # todo
+            self.additional_attributes['Signer'] = publisher
+            # todo what if both signer and publisher are found? return list? concatenate?
```
Comment (David-BMS) [inline Packs/MalwareParser/Scripts/Parse/MalwareFindings.py:147]: I guess the name is confusing 
the intention is of course who signed the file 
and that we see in File.DigitalSignature.Publisher

File: Packs/MalwareParser/Scripts/Parse/MalwareFindings.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+                         search_value=ThreatIntel.KillChainPhases.COMMAND_AND_CONTROL)
+
+
+class FileSignature(MalwareFinding):
+    def _parse_context(self) -> None:
+        self.additional_attributes = {}
+        self.status = Status.SUSPICIOUS  # default
+
+        # MSDE
+        self._parse_single('MicrosoftATP.File.IsValidCertificate', 'true', Status.BENIGN, Source.TODO)  # todo True?
```
Comment (David-BMS) [inline Packs/MalwareParser/Scripts/Parse/MalwareFindings.py:136]: Yes if signed true

File: Packs/MalwareParser/Scripts/Parse/MalwareFindings.py [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+    def __init__(self, name: str, context: dict, question: Optional[str] = None):
+        self.name: name
+        self.context = context
+        self.question = question
+        self._parse_context()
+
+    def to_context(self) -> dict:
+        context = {
+            'Source': [source.value for source in self.sources],  # todo Source or Sources key?
```
Comment (David-BMS) [inline Packs/MalwareParser/Scripts/Parse/MalwareFindings.py:39]: Sources makes more sense

File: Packs/MalwareParser/pack_metadata.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    "email": "",
+    "categories": [
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [],
+    "useCases": [],
+    "keywords": [],
+    "marketplaces": [
+        "xsoar",
+        "marketplacev2"
```
Comment (David-BMS) [inline Packs/MalwareParser/pack_metadata.json:17]: Should be just XSOAR

File: Packs/Malware/Scripts/FindingsParse/FindingsParse.py [hunk 5]
```diff
@@ -0,0 +1,237 @@
+import traceback
+from abc import ABC, abstractmethod
+from enum import Enum
+from typing import Optional, Dict, Set, Union
+
+import demistomock as demisto
+
+try:
+    from CommonServerPython import ThreatIntel, argToBoolean, CommandResults, return_results, return_error, Common
+except:  # todo remove
```
Comment (DeanArbel) [inline Packs/Malware/Scripts/FindingsParse/FindingsParse.py]: ahem

File: Packs/Malware/Scripts/FindingsParse/FindingsParse.py [hunk 6]
```diff
@@ -0,0 +1,237 @@
...
+        PrivilegeEscalation(context),
+        CommandAndControl(context),
+    )]
+
+
+def main():
+    try:
+        context = demisto.callingContext.get('context', {}).get('ExecutionContext', {})
+        # context = json.load(open('test_data/dummy_json.json'))  # todo remove
+        result = parse_command(context)  # todo remove
```
Comment (DeanArbel) [inline Packs/Malware/Scripts/FindingsParse/FindingsParse.py]: todo?

File: Packs/Malware/Scripts/FindingsToTable/FindingsToTable.py [hunk 7]
```diff
@@ -0,0 +1,61 @@
+import traceback
+from enum import Enum
+
+
+# Enable for development
+# from CommonServerPython import CommandResults, return_results, return_error
+# import demistomock as demisto
```
Comment (DeanArbel) [inline Packs/Malware/Scripts/FindingsToTable/FindingsToTable.py:7]: ?

**Review by DeanArbel — COMMENTED** [review]
Please remove comments / todos.

File: Packs/Malware/Scripts/FindingsParse/FindingsParse.py (hunk 5 above)
Comment (dorschw) [reply Packs/Malware/Scripts/FindingsParse/FindingsParse.py]: already removed 😇

File: Packs/Malware/Scripts/FindingsParse/FindingsParse.py (hunk 6 above)
Comment (dorschw) [reply Packs/Malware/Scripts/FindingsParse/FindingsParse.py]: all these are gone

**Review by DeanArbel — APPROVED** [review]

**Comment by julieschwartz18** [comment]
Doc review complete for the MalwareFindingsToTable.yml file. @dorschw - please review the updates and confirm they are ok.

**Pull request merged**